src/ott/solvers/linear/sinkhorn.py

```diff
@@ -42,6 +44,21 @@
 
 __all__ = ["Sinkhorn", "SinkhornOutput", "solve"]
 
+ProgressCallbackFn = Callable[
+    [Tuple[np.ndarray, np.ndarray, np.ndarray, NamedTuple]], None]
+"""
```

Comment:
This docstring is really helpful, but not visible in the online docs, should be moved to the argument description in `Sinkhorn`.